### DISCUSSION TITLE
CLI for dumping .DAT from .PARK

### DISCUSSION THREAD

**Discussion by Richard-L** [OpenRCT2/OpenRCT2#15968] [Feature requests]
@duncanspumpkin advised to post this here: It would be fantastic if there was a way to dump CSO from the new .PARK files, similar to how we can extract sprite data from objects.

I understand .PARK can bundle both old .DAT as well as the new .PARKOBJ files, correct? If so it would be fantastic if it could dump both.

Many thanks for considering.

**Comment by duncanspumpkin** [comment]
I was thinking could dump out all the packed parkobjs and dats and also a file detailing all identifier names that are in use in the file. Then you can easily confirm that it is using purely ncso.

**Reply by Richard-L** [reply]
Would be fantastic
